mysql-test/suite/sys_vars/r/all_vars.result

```diff
@@ -15,6 +15,8 @@ left join t1 on variable_name=test_name where test_name is null ORDER BY variabl
 There should be *no* variables listed below:
 DISABLED_STORAGE_ENGINES
 DISABLED_STORAGE_ENGINES
+ENCRYPT_BINLOG
```

Comment:
sys_vars.all_vars should not pass like that if sys_vars.encrypt_binlog_basic is now present

Reply:
Right, done